ArcadiaParties.API/Controllers/UsersController.cs

```diff
@@ -0,0 +1,53 @@
+﻿using ArcadiaParties.CQRS.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Swashbuckle.AspNetCore.Annotations;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace ArcadiaParties.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {        
+        private readonly IMediator _mediator;
+
+        public UsersController(IMediator mediator)
+        {
+            _mediator = mediator;
+        }
+
+        [ProducesResponseType(StatusCodes.Status200OK)]
```

Comment:
Also you need to specify return type if any.

Reply:
Return type annotation added